### PR TITLE
feat/SOF-6914

### PR DESCRIPTION
**Pull request by seankwarren**
Includes a custom implementation of MUI text input and select with default values

### CODE REVIEW

File: src/mui/components/textField/BasicTextField.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        InputLabelProps: inputLabelProps,
+        onChange: (e) => onChange(e.target.value),
+    };
+
+    if (value !== undefined) {
+        textFieldProps.value = value;
+    }
+
+    if (defaultValue !== undefined) {
+        textFieldProps.defaultValue = defaultValue;
```
Comment (zoomchik) [inline src/mui/components/textField/BasicTextField.tsx]: Where is textFieldProps used?

File: src/mui/components/select/BasicSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+type BasicSelectProps = {
+    disabled?: boolean;
+    selectedValue: string;
+    options: AvailableValue[];
+    onChange: (newValue: string) => void;
+    label: string;
+    labelAsPlaceholder?: boolean;
+    renderMenuItemContent?: (value: AvailableValue) => JSX.Element;
+};
+// TODO: move to cove.js
```
Comment (zoomchik) [inline src/mui/components/select/BasicSelect.tsx]: Remove

File: src/mui/components/textField/BasicTextField.tsx (hunk 1 above)
Comment (seankwarren) [reply src/mui/components/textField/BasicTextField.tsx]: cleaned up

File: src/mui/components/select/BasicSelect.tsx (hunk 2 above)
Comment (seankwarren) [reply src/mui/components/select/BasicSelect.tsx]: removed

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**